static/js/default-map-api-key.js

```diff
@@ -13,7 +13,7 @@ export function getDefaultMapApiKey(mapProvider) {
       case 'google':
         return 'AIzaSyB5D45ghF1YMfqTLSzWubmlCN1euBVPhFw';
       case 'mapbox':
-        return 'pk.eyJ1IjoieWV4dCIsImEiOiJqNzVybUhnIn0.hTOO5A1yqfpN42-_z_GuLw';
+        return process.env.MAPBOX_KEY;
```

Comment:
could this break existing implementations if they try to upgrade the HH theme? if so, we might need additional documentation around this release

Reply:
this will definitely break people, we may need to get an approval

Reply:
or work with the pages team to move this in platform

Reply:
I thought there was some work done around locking down mapbox keys and making sure each account supplies their own. was that only for locators, or also for search frontends like this?

Reply:
discussed in offline slack thread, reverting this change since it will likely break for existing customers